blob/blob.go

```diff
@@ -97,14 +99,14 @@ func NewBlob(shareVersion uint8, namespace share.Namespace, data []byte) (*Blob,
 		return nil, err
 	}
 
-	blob := tmproto.Blob{
+	blob := blob.Blob{
 		NamespaceId:      namespace.ID(),
 		Data:             data,
 		ShareVersion:     uint32(shareVersion),
 		NamespaceVersion: uint32(namespace.Version()),
 	}
 
-	com, err := types.CreateCommitment(&blob)
+	com, err := inclusion.CreateCommitment(&blob, merkle.HashFromByteSlices, appconsts.SubtreeRootThreshold(v1.Version))
```

Comment:
[for reviewers] Invoking with `v1.Version` behaves identically to invoking with `v2.Version` because the implementation of this function doesn't actually care about the version that is provided as an argument.

See https://github.com/celestiaorg/celestia-app/blob/c0904490d3e60a8206b43728378b4ae4c097edd1/pkg/appconsts/versioned_consts.go#L21-L23